MAUI/TextInputLayout/Migration.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfTextInputLayout | Syncfusion 
+description: Learn all about Migrating from Syncfusion Xamarin TextInputLayout to Syncfusion .NET MAUI TextInputLayout control and more here.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---  
+
+# Migrating from Xamarin TextInputLayout to .NET MAUI TextInputLayout 
+
+To migrate easier from Xamarin SfTextInputLayout to [.NET MAUI SfTextInputLayout](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfTextInputLayout.html), we kept most of the APIs from Xamarin SfTextInputLayout in MAUI SfTextInputLayout. However, to maintain the consistency of API naming in MAUI SfTextInputLayout, we renamed some of the APIs. The APIs changed in MAUI SfTextInputLayout from Xamarin SfTextInputLayout are detailed as follows:
```

Comment:
To migrate easier from Xamarin SfTextInputLayout to [.NET MAUI SfTextInputLayout](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfTextInputLayout.html), we kept most of the APIs from Xamarin SfTextInputLayout in MAUI SfTextInputLayout. However, to maintain the consistency of API naming in MAUI SfTextInputLayout, we renamed some of the APIs. The APIs changed in MAUI SfTextInputLayout from Xamarin SfTextInputLayout as detailed in the following:

Reply:
Addressed the changes